### PR TITLE
Fix kserve local testing env

### PR DESCRIPTION
**Pull request by yuzisun**
**What this PR does / why we need it**:
`make test` was failing locally as the crd paths are different for tests from different packages, we should pass the crd path to the `SetupEnvTest` function instead of enumerating all the possible paths there.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Type of changes**
Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Feature/Issue validation/testing**:

Please describe the tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- Logs

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Checklist**:

- [ ] Have you added unit/e2e tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

**Release note**:

```release-note

```

**Re-running failed tests**

- `/rerun-all` - rerun all failed workflows.
- `/rerun-workflow <workflow name>` - rerun a specific failed workflow. Only one workflow name can be specified. Multiple /rerun-workflow commands are allowed per comment.

### CODE REVIEW

File: pkg/openapi/swagger.json [hunk 1]
```diff
@@ -1169,7 +1169,7 @@
       "type": "object",
       "properties": {
         "annotations": {
-          "description": "Annotations that will be add to the component pod. More info: http://kubernetes.io/docs/user-guide/annotations",
+          "description": "Annotations that will be added to the component pod. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/",
```
Comment (sivanantha321) [inline pkg/openapi/swagger.json:1172]: Wondering how was the CI didn't complain about this ? Maybe not rebased ? Should we have a check for rebase ?

Comment (yuzisun) [reply pkg/openapi/swagger.json:1172]: I think the merged PR probably was not rebased and it was created before the verification was added

Comment (sivanantha321) [reply pkg/openapi/swagger.json:1172]: Oh Okay. Also I am thinking we can add a auto retry mechanism for Infra setup actions.https://github.com/marketplace/actions/retry-action. WDYT ?

Comment (yuzisun) [reply pkg/openapi/swagger.json:1172]: Sounds good

**Pull request merged**